packages/lab/src/ProgressBar/ProgressBar.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import PropTypes from "prop-types";
+import clsx from "clsx";
+
+import clamp from "lodash/clamp";
+import { HvTypography } from "@hitachivantara/uikit-react-core";
+
+import { withStyles } from "@material-ui/core";
+
+import styles from "./styles";
+
+/**
+ * ProgressBar provides feedback about a process that is taking place in the application.
+ */
+const HvProgressBar = (props) => {
+  const { className, classes, value = 0, error, undeterminate = false, ...others } = props;
```

Comment:
error -> status:` invalid, valid, in progress ; "success", "in progress", "error"`. 

add a labelProp to spread generic values on the label of the element
should be typed as generic object in js
should be typed as div object in ts